walkthroughs/week-3-server/examples/server-stats/pom.xml

```diff
@@ -29,6 +29,12 @@
       <artifactId>gson</artifactId>
       <version>2.8.6</version>
     </dependency>
+
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.6</version>
+    </dependency>
```

Comment:
I think you can remove the changes to your examples files from your PR.

Reply:
Done!